src/components/chapter/chapter.js

```diff
@@ -63,34 +80,25 @@ function Chapter({ id, theme, title, image, images, description, currentChapterI
     setCurrentChapter(id);
     setCurrentAction('enter');
   };
-  const onLeave= () => {
+  const onLeave = () => {
     setCurrentChapter(id);
     setCurrentAction('leave');
   };
 
   return (
     <div id={id} className={cx(classList, ALIGNMENTS[alignment])}>
-      <Waypoint
-        onEnter={onEnter}
-        onLeave={onLeave}
-      />
-      <div className={theme}>
-        {images &&
-          images.filter((i) => i.position === 'top').map((i) => renderImage(i))}
+      <Waypoint onEnter={onEnter} onLeave={onLeave} />
+      <div className={cx(theme, 'rounded-lg')}>
+        {images && images.filter((i) => i.position === 'top').map((i) => renderImage(i))}
         {title && (
           <div className="content text-base py-12 px-12 leading-6">
-            {title && <h3 className="font-serif text-2xl pb-6">{t(title)}</h3>}
-            {description && (
-              <p className="text-sm leading-8">{t(description)}</p>
-            )}
+            {title && <h3 className="font-lora text-2xl pb-6">{t(title)}</h3>}
```

Comment:
the title text is 32px height and has also 32px line height could you adjust this or change the 2xl definition? Also the pb should be 4 or 5